src/psyclone/nemo.py

```diff
@@ -48,7 +48,8 @@
 from psyclone.psyGen import PSy, Invokes, Invoke, InvokeSchedule, \
     InlinedKern
 from psyclone.errors import InternalError
-from psyclone.psyir.nodes import Loop, Schedule
+from psyclone.psyir.nodes import Loop, Schedule, Reference
```

Comment:
Unused Reference and SymbolError imports

Reply:
Oops.